Rework SD Card Setup to be inclusive of some edge cases (#95)
cfw-guide/dsi.cfw.guide: _pages/en_US/sd-card-setup.md

   <h2>Windows</h2>
</noscript>

## Section I - Formatting your SD card with SD Formatter

This section formats the SD card to the specifications by the SD Card Association. This can fix many issues that may occur with running homebrew applications.
{: .notice--info}

Any 64GB or larger SD cards will be formatted to `exFAT` in this process. You _must_ follow Section II to re-format to `FAT32`.
{: .notice--danger}

1. Download the latest version of [SD Formatter](https://www.sdcard.org/downloads/formatter/sd-memory-card-formatter-for-windows-download/)
  - Accept the End User License Agreement to start the download
1. Run `SD Card Formatter` (the `.exe` file) in the downloaded `.zip` file with Adminstrator privileges, then install the program

>   - The version number may be different, but it will install the same program

Reviewer: Has a version number been mentioned? This line seems kinda random as I don't see a version being mentioned anywhere

Author: The .exe file has a version number. I actually tried to rewrite the line trying to get around it, and maybe I succeeded?

Reviewer: I would maybe change it to `SD Card Formatter Setup` but otherwise yeah I think that works fine.

1. Take note of the device mount point. In our example above, it was `mmcblk0p1`
   - If `RO` is set to 1, make sure the lock switch is not slid down
1. Hit CTRL + C to exit the menu
1. Follow the instructions relevant to your SD card's capacity:

>    - 2GB or lower: Type in `sudo mkdosfs /dev/(device mount point from above) -s 64 -F 16` to create a single FAT16 partition with 32 KB cluster size on the SD card
>    - 4GB or higher: Type in `sudo mkdosfs /dev/(device mount point from above) -s 64 -F 16` to create a single FAT32 partition with 32 KB cluster size on the SD card

Reviewer: These are the same command, is it supposed to be this?
```suggestion
   - 2GB or lower: Type in `sudo mkdosfs /dev/(device mount point from above) -s 64 -F 16` to create a single FAT16 partition with 32 KB cluster size on the SD card
   - 4GB or higher: Type in `sudo mkdosfs /dev/(device mount point from above) -s 64 -F 32` to create a single FAT32 partition with 32 KB cluster size on the SD card
```

   <h2>macOS</h2>
</noscript>

## Section I - Formatting your SD card with SD Formatter

This section formats the SD card to the specifications by the SD Card Association. This can fix many issues that may occur with running homebrew applications.
{: .notice--info}

Any 64GB or larger SD cards will be formatted to `exFAT` in this process. You _must_ follow Section II to re-format to `FAT32`.
{: .notice--danger}

> 1. Download the latest version of [SD Formatter](https://www.sdcard.org/downloads/formatter/sd-memory-card-formatter-for-windows-download/)

Reviewer: ```suggestion
1. Download the latest version of [SD Formatter](https://www.sdcard.org/downloads/formatter/sd-memory-card-formatter-for-mac-download/)
```

  - Accept the End User License Agreement to start the download

> 1. Run `SD Card Formatter` (the `.exe` file) in the downloaded `.zip` file with Adminstrator privileges, then install the program
>   - The version number may be different, but it will install the same program
> 1. Run `SD Card Formatter` from the Start Menu with Adminstrator privileges

Reviewer: ```suggestion
1. Run `Install SD Card Formatter` (the `.mpkg` file) in the downloaded `.zip` file
  - The version number may be different, but it will install the same program
1. Run `SD Card Formatter`
```
